test/Sentry.Unity.Tests/UnitySentryOptionsTests.cs

```diff
@@ -34,24 +37,39 @@ public void Options_WriteRead_Equals()
                 Debug = true,
                 DebugOnlyInEditor = false,
                 DiagnosticLevel = SentryLevel.Info,
-                RequestBodyCompressionLevel = CompressionLevelWithAuto.NoCompression,
                 AttachStacktrace = true,
                 SampleRate = 1f,
-                Release = "release",
-                Environment = "test"
             };
 
-            // act
-            using var memory = new MemoryStream();
-            using var writer = new Utf8JsonWriter(memory);
-            optionsExpected.WriteTo(writer);
+            var scriptableSentryUnity = ScriptableObject.CreateInstance<ScriptableSentryUnityOptions>();
+            scriptableSentryUnity.Enabled = optionsExpected.Enabled;
+            scriptableSentryUnity.Dsn = optionsExpected.Dsn;
+            scriptableSentryUnity.CaptureInEditor = optionsExpected.CaptureInEditor;
+            scriptableSentryUnity.Debug = optionsExpected.Debug;
+            scriptableSentryUnity.DebugOnlyInEditor = optionsExpected.DebugOnlyInEditor;
+            scriptableSentryUnity.DiagnosticLevel = optionsExpected.DiagnosticLevel;
+            scriptableSentryUnity.AttachStacktrace = optionsExpected.AttachStacktrace;
+            scriptableSentryUnity.SampleRate = (float)optionsExpected.SampleRate;
 
-            var jsonRaw = Encoding.UTF8.GetString(memory.ToArray());
-            using var jsonDocument = JsonDocument.Parse(jsonRaw);
-            var optionsActual = SentryUnityOptions.FromJson(jsonDocument.RootElement);
+            // Act
+            var optionsActual = ScriptableSentryUnityOptions.LoadFromScriptableObject(scriptableSentryUnity);
 
-            // assert
-            AssertOptions(optionsActual, optionsExpected);
+            AssertOptions(optionsExpected, optionsActual);
+        }
+
+        [Test]
+        public void Options_JsonAndScriptable_Equal()
+        {
+            var jsonTextAsset = new TextAsset(File.ReadAllText(GetTestOptionsFilePath()));
+            var scriptableOptions = ScriptableObject.CreateInstance<ScriptableSentryUnityOptions>();
+
+            var expectedOptions = JsonSentryUnityOptions.LoadFromJson(jsonTextAsset);
+
+            // Act
```

Comment:
probably should be removed or arrange, assert be added

Reply:
```suggestion
```